invoices/invoiceregistry.go

```diff
@@ -682,13 +686,30 @@ func (i *InvoiceRegistry) processKeySend(ctx invoiceUpdateCtx) error {
 		return errors.New("final expiry too soon")
 	}
 
+	// In order to promote a legacy keysend into an AMP payment, we'll start
```

Comment:
👍

Reply:
What was the rationale again for promoting the legacy keysends? Instead of leaving the legacy logic unchanged and adding a new block to jit-insert amp invoices?

Reply:
Discussed offline. No invoices can be created anymore without payment address. Maybe good to mention in the comment?